src/main/typescript/components/overview/DepositTableRow.tsx

```diff
@@ -15,13 +15,8 @@
  */
 import * as React from "react"
 import * as dateFormat from "dateformat"
-import { DeleteState, Deposit, DepositId, DepositState } from "../../model/Deposits"
-import { Link } from "react-router-dom"
-import { depositFormRoute } from "../../constants/clientRoutes"
-
-function isEditable({ state }: Deposit): boolean {
```

Comment:
if I understand you correctly, you want to determine whether a row is editable by the above component (a.k.a. the `DepositOverview`)?

👍

that way we can keep this component as 'dumb' as possible.